control_panel_api/models.py

```diff
@@ -14,6 +14,7 @@
 class User(AbstractUser):
     auth0_id = models.CharField(max_length=128, primary_key=True)
     name = models.CharField(max_length=256, blank=True)
+    verified_email = models.BooleanField(default=False)
```

Comment:
Annoying comment on this field name. Wouldn't make more sense to call this `email_verified`? That way would be clear for the name is flag to check if the email was verified instead of an email which is verified.

Also, if both fields start with `email` is clear they're very related.

...said that, you probably already wrote some FE code going with this so probably annoying to change so ignore it if it's too much hassle and you reckon is not worth it.

Reply:
Yep, will change